integration-tests/src/test/resources/oke/terraform/cluster.tf

```diff
@@ -0,0 +1,63 @@
+/*
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+*/
+variable "cluster_kubernetes_version" { default = "v1.11.5" }
```

Comment:
This is a no longer supported Kubernetes version.

Reply:
I will correct default values but I provide values for variables via tfvar file